tests/support/napalm.py

```diff
@@ -391,7 +391,7 @@ def get_environment(self):
     def get_arp_table(self):
         return TEST_ARP_TABLE
 
-    def get(self, key, default=None, *args, **kwargs):
+    def get(self, key, *args, default=None, **kwargs):
```

Comment:
I don't think this is gonna work. `*args` and `**kwargs` go at the end.

Reply:
It will work on modern Python. Considering this is going into develop, I believe it should be fine, though I'd have to look it up to be sure when it was added to Python.

Reply:
It works

Reply:
Yea @nodebanger you'r right it does not work in python2 :( but you can add a commit that will disable the pylint warning